src/main/java/com/crowdin/cli/commands/Commands.java

```diff
@@ -585,12 +585,16 @@ private void uploadSources(boolean autoUpdate) {
                                         .map(fileEntity -> {
                                             String fileId = fileEntity.getId().toString();
                                             String projectId = projectInfo.getProjectId();
-                                            Map<String, Integer> schemeObject = getSchemeObject(file);
-                                            String updateOption = getUpdateOption(file);
-                                            Integer escapeQuotes = (int) file.getEscapeQuotes();
-                                            RevisionPayload revisionPayload = new RevisionPayload(storageId, schemeObject, file.getFirstLineContainsHeader(), updateOption, escapeQuotes);
 
-                                            return new RevisionsApi(this.settings).createRevision(projectId, fileId, revisionPayload).execute();
+                                            UpdateFilePayload updateFilePayload = new UpdateFilePayload() {{
```

Comment:
Better to have separate class which will extend UpdateFilePayload and have constructor which will consume storageId,export and import options